### ISSUE TITLE
Accessible Alt Attributes

### ISSUE THREAD

**Issue by mkng1** [mkng1/Challenge1#4]
Alt Attributes should be added to relevant elements for accessibility

All image elements in the document are generic marketing images without the need for description.

Therefore null alt attributes have been added where necessary
